internal/pkg/template/templates/workloads/partials/cf/service-base-properties.yml

```diff
@@ -21,6 +21,17 @@ DeploymentConfiguration:
     AlarmNames: {{ .DeploymentConfiguration.RollbackAlarms }}
     Enable: true
     Rollback: true
+  {{- else if or .DeploymentConfiguration.CPUUtilization .DeploymentConfiguration.MemoryUtilization }}
+  Alarms:
+    AlarmNames:
```

Comment:
I can see only `AlarmNames` is changing in both the scenarios i.e. basic and advanced. So can we club them here to reduce the code duplicity?

```
  {{- if or .DeploymentConfiguration.RollbackAlarms .DeploymentConfiguration.CPUUtilization .DeploymentConfiguration.MemoryUtilization }}
  Alarms:
    AlarmNames: {{- if .DeploymentConfiguration.RollbackAlarms }}{{ .DeploymentConfiguration.RollbackAlarms }}
    {{ - else if .DeploymentConfiguration.CPUUtilization }}
    - !Sub ${AWS::StackName}-CPUAlarm
    {{- else }}
    - !Sub ${AWS::StackName}-MemAlarm
    {{- end }}
    Enable: true
    Rollback: true
  {{- end }}
```

Reply:
We might also write a `DeploymentRollbackAlarmsEnabled` function for the `{{- if or .DeploymentConfiguration.RollbackAlarms .DeploymentConfiguration.CPUUtilization .DeploymentConfiguration.MemoryUtilization }}`

Reply:
Thanks! I did a version of this that isn't XOR and while has an extra line for the sake of readability.